tools/link.py

```diff
@@ -2403,9 +2424,9 @@ def modularize():
   return moduleRtn;
 }
 ''' % {
-    'maybe_async': async_emit,
-    'src': src,
-  }
+      'maybe_async': async_emit,
+      'src': src,
+    }
```

Comment:
Is this indentation better than before?  Maybe revert this?

Reply:
Flake wants this because of the new if/else above.